shared/common-adapters/input.native.js

```diff
@@ -116,10 +116,15 @@ class Input extends Component<void, Props, State> {
     if (this.props.onEnterKeyDown && e.key === 'Enter') {
       this.props.onEnterKeyDown(e)
     }
+
+    if (this.props.onEscapeKeyDown && e.key === 'Escape') {
+      this.props.onEscapeKeyDown(e)
```

Comment:
so i'm advocating for removing changes to input and instead here https://github.com/keybase/client/pull/8100/files#diff-0aa3c05310e25cf6a99c0caee30ce1b7R71
just have the e.key === 'Escape' logic